src/Maker/MakeController.php

```diff
@@ -53,26 +54,42 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
 
     public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
     {
+        $this->projectDir = $command->getApplication()->getKernel()->getContainer()->getParameter('kernel.project_dir');
     }
 
     public function getParameters(InputInterface $input): array
     {
         $controllerClassName = Str::asClassName($input->getArgument('controller-class'), 'Controller');
         Validator::validateClassName($controllerClassName);
 
+        $title = "Hello {$controllerClassName}!";
+        if (file_exists($this->projectDir.'/templates/base.html.twig')) {
```

Comment:
However, I think what you need here is the `%twig.default_path%` value, later should be `$this->templateDir` instead?

Reply:
We cannot use `twig.default_path` because the parameter is not always defined.
But anyway, just targeting the default dir structure is fine IMHO.

Reply:
The bundle is intended to work with Symfony 3.4 or newer, right? and the `twig.default_path` parameter was introduced since 3.4, so I guess this should work?

Anyway at least we can avoid to hardcode the `templates/` directory with something like this?:
```
<argument type="expression">container.hasParameter('twig.default_path') ? parameter('twig.default_path') : parameter('kernel.project_dir') ~ '/app/Resources/views'</argument>
```

Reply:
It's not a matter of version: the param exists only if TwigBundle is.
I think this is not worth pursuing: `templates/` is what the default recipe provides.
There are already many hardcoded, eg the `src/` folder. This is legitimate for the purpose.
The day this statement will become false, someone could try a PR, fixing all hardcoded things at one. Dealing with the required complexity is not needed now, if anytime ever.
My 2cts:)

Reply:
Sorry, I thought I commented on this line, but I apparently did not! I believe we should use `FileManager::fileExists()` - it's aware of the root directory and I want to keep all the path stuff going through this